src/api/schema/components/source.rs

```diff
@@ -90,8 +92,8 @@ impl Source {
     }
 
     /// Source output type
-    pub async fn output_type(&self) -> SourceOutputType {
-        self.get_output_type()
+    pub async fn output_type(&self) -> Vec<SourceOutputType> {
```

Comment:
Since this now returns an array, this should probably be renamed to `output_types`, with the doc comment updated to reflect.

This is a breaking change in the GraphQL API, but returning an array already breaks it, so this is somewhat equivalent.